etcdserver/server.go

```diff
@@ -1002,6 +1002,26 @@ func (s *EtcdServer) LeaderStats() []byte {
 
 func (s *EtcdServer) StoreStats() []byte { return s.store.JsonStats() }
 
+func (s *EtcdServer) checkMembershipOperationPermission(ctx context.Context) error {
```

Comment:
Allowing `EtcdServer.authStore == nil` is ok for me but it requires many changes because the callers of `EtcdServer.AuthStore()` must be changed and there are more than 20 callers. Is it ok?